otevírací-doba/draft/index.html

```diff
@@ -394,7 +394,7 @@ <h3>Komplexní otevírací doba</h3>
                    Příklad a schéma jsou stále ve vývoji.
                 </p>
 
-                <aside class="example" title="Komplexní oznámení v JSON">
+                <aside class="example" title="Oznámení v JSON">
```

Comment:
Proč má otevírací doba zase "Oznámení v JSON"?

Reply:
@spalrobert prosím o vyjádření, možná jsem to zvojtil já toto. Změnil jsem to komplexní na oznámení, protože tam je jen jedno.

Reply:
@spalrobert Oznámení je tam od dob co to bylo zkopírováno z oznámení na úřední desce. Čili by to mělo být Otevírací doba v JSON, nikoliv Oznámení v JSON

Reply:
oprava viz [https://github.com/jirkamarekbrno/otevrene-formalni-normy/commit/94ed4d2bcb38c63d150873850358b83380a1c683](url)